libs/asset-viewer/src/lib/components/map/map.component.ts

```diff
@@ -198,6 +204,10 @@ export class MapComponent implements AfterViewInit, OnChanges, OnDestroy {
     }
   }
 
+  private equalsCheck<T>(a: T[] | null, b: T[] | null) {
```

Comment:
Since this is generic it coiuld live inside its own file as it does not really have anything to do withthe map per se